core/domain/story_domain.py

```diff
@@ -659,6 +661,7 @@ def __init__(
             thumbnail_filename: str|None. The thumbnail filename of the story.
             thumbnail_bg_color: str|None. The thumbnail background color of
                 the story.
+            thumbnail_size_in_bytes: int|None. The size of thumbnail in bytes.
```

Comment:
it should be `int|0`, will fix this

Reply:
None is a better option, after discussing with Srikanth, get to know about this and this will also help in handling errors.